tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+from keras import backend as K
+
+
+@pytest.fixture(autouse=True)
+def clear_session_after_test():
+    yield

```

Comment:
Wouldn't this be problematic for tests that have arguments?

Reply:
I don't think so. It's quite a common pattern in pytest. See this example from the docs: https://docs.pytest.org/en/latest/fixture.html#autouse-fixtures-xunit-setup-on-steroids